access/email/config.go

```diff
@@ -183,10 +189,29 @@ func (c *Config) CheckAndSetDefaults() error {
 		c.Log.Severity = "info"
 	}
 
-	// Validate emails in user aliases
-	for _, e := range c.Delivery.Recipients {
-		if !lib.IsEmail(e) {
-			return trace.BadParameter("Invalid email address %v in users.recipients", e)
+	if len(c.Delivery.Recipients) > 0 {
+		if len(c.RoleToRecipients) > 0 {
+			return trace.BadParameter("provide either delivery.recipients or role_to_recipients, not both.")
```

Comment:
Nit: Don't put dots at the end of error messages. Same below.
```suggestion
			return trace.BadParameter("provide either delivery.recipients or role_to_recipients, not both")
```